collector/collector.lua

```diff
@@ -28,19 +29,20 @@ sorted_bags = L{'safe', 'safe2', 'storage', 'locker',
                 'slip 31', 'key items', }
 
 
-function add_result(result,bag,count,augments)
+function add_result(result,bag,count,augments,lvl)
     local count = count > 1 and ' ('..count..')' or ''
+    local level = lvl == nil and '' or ' %s':format(lvl)
     local rank = augments ~= nil and augments.rank ~= nil and ' (Rank: %s)':format(augments.rank) or ''
-    return (bag == 'missing' and result:color(259) or result:color(258)) .. count .. rank
+    return (bag == 'missing' and result:color(259) or result:color(258)) .. count .. level .. rank
 end
```

Comment:
Not a huge deal but checking for a variable to be nil is a little roundabout, I'd turn around the level check for consistency with the other value.

Reply:
True.. the consistency with the rest of the code is better.  Fixed in [a84395a](https://github.com/lili-ffxi/FFXI-Addons/pull/13/commits/a84395a2c2d509121138be33ee702a5be90401bc)